packages/x-components/static-docs/experience-search-and-discovery/README.md

```diff
@@ -130,3 +103,29 @@ your native app search solution.
     'x_empathize',
     'x_results',]"
 />
+
+::: warning
+
+Some X&nbsp;Components are supported by Empathy Platform search and discovery features. If you’re
+using a different search engine, make sure it supports all the functionalities you need.
+
+:::
+
+## Try X Components to…
+
+- Surprise your shoppers by **rendering shopping-aimed search and discovery features** that
+  intuitively guide and inspire them with trends, suggestions, and recommendations, based on your
+  shoppers’ behavior.
+- Add a **predictive layer** to your search UI that gathers all search functionalities in one, from
+  autocomplete and autosuggest to ideas of what to look for next.
+- Delight your shoppers with an **elegant and multi-configurable SERP** to help them discover what
+  they’re exactly looking for.
```

Comment:
```suggestion
- Delight your shoppers with an **elegant and multi-configurable SERP**, helping them to discover what
  they’re exactly looking for.
```

Reply:
- ING form: we want to express the goal or objective, so it's better to use a to + inf clause here.
- "help + inf verb" & "help + to + inf verb" --> both options are correct :)